tests/next_tests/integration_tests/feature_tests/ffront_tests/test_import_from_mod.py

```diff
@@ -0,0 +1,36 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from tests.next_tests.integration_tests.feature_tests.ffront_tests import dimensions_mod as dims
+from tests.next_tests.integration_tests.cases import IDim, KDim
+import gt4py.next as gtx
+import numpy as np
+from tests.next_tests.integration_tests import cases
+from tests.next_tests.integration_tests.cases import cartesian_case
+
+from tests.next_tests.integration_tests.feature_tests.ffront_tests.ffront_test_utils import (
+    exec_alloc_descriptor,
+    mesh_descriptor,
+)
+
+
+def test_import_dims_module(cartesian_case):
+    @gtx.field_operator
+    def mod_op(f: cases.IKField) -> cases.IKField:
+        return f
+
+    @gtx.program
+    def mod_prog(f: cases.IKField, out: cases.IKField):
+        mod_op(f, out=out, domain={dims.IDim: (0, 8), dims.KDim: (0, 3)})
```

Comment:
```suggestion
        mod_op(f, out=out, domain={cases.IDim: (0, 8), cases.KDim: (0, 3)})
```
otherwise, the definitions in `cases` for `IKField` might not match the dimensions defined in `dimensions_mod`. If you want to isolate the dimensions in a module you could probably define `dimensions_mod` as

```
from tests.next_tests.integration_tests import cases

IDim = cases.IDim
KDim = cases.KDim
```